src/ui/base/resource/resource_bundle.cc

```diff
@@ -75,6 +75,11 @@ ResourceBundle* g_shared_instance_ = NULL;
 
 base::FilePath GetResourcesPakFilePath(const std::string& pak_name) {
   base::FilePath path;
+#if defined(USE_CBE)
+  if (base::PathService::Get(base::DIR_EXE, &path))
+    return path.AppendASCII(pak_name.c_str());
+#endif // defined(USE_CBE)
+
```

Comment:
This is wrong. CBE resources (basically almost anything that is not in chrome/) should be fetched in common assets path. What you need to fix is the cbe installation to install this in the shared cbe path, not in /opt/chromium68

Reply:
Ok, agree this is not the correct way to fix the issue for both chrome and libcbe.
I understand non-chrome resources should be in a common assets path (some of them are already being installed to `/usr/lib/cbe/`). On the other hand, what happens with `//chrome` resources (e.g: `chrome_100_percent.pak`)? Should they also go to this same location? I mean, in this case they are not `libcbe` assets, right?

Reply:
Actually code that loads chrome_100_percent.pak is in ui/. But even if it was in chrome/ take into account 99% of chrome is part of libcbe.so, so you could put them in cbe shared path instead of chrome bin path.

Basically, if we moved assets to point to bindir we'd have to make all the assets be there, or start adding a CBE_ASSETS path and move there the ones that would make sense. I prefer the simplest solution (all ASSETS to cbe shared path).

Reply:
Ok. Just pushed the changes as per your request.
For now installing only `chrome_100_percent.pak` to install list.